Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -732,16 +732,17 @@ public abstract class Sequence<T>: ISequence<T>
       return hashCode;
     }
     public override string ToString() {
```

Comment:
This reminds me we should do something to remove the two checked-in copies of DafnyRuntime.cs.

Reply:
Agreed, I wasn't sure if that was safe to remove Binaries and add it to .gitignore.

Reply:
I think it is. The file needs to be part of the binary distribution (until we ship a DLL for the DafnyRuntime instead), but the build is set up to copy the .cs file to Binaries anyway.